.github/chainguard/neko.sts.yaml

```diff
@@ -0,0 +1,13 @@
+issuer: https://accounts.google.com
+# neko77@neko77.iam.gserviceaccount.com
+subject: "113809027484440661352"
+permissions:
+  discussions: read
+  email: read
```

Comment:
I'm adding all of the capabilities except this to the App's permission set now.  The only place I see email is on a section that looks like it's for UATs vs. IATs, and we only use IATs.  Can we try it without this and iterate?